install/connections.csv

```diff
@@ -197,6 +197,7 @@ ETVZ,30,East Tongariro,,3,2014-03-11T00:01:00Z,9999-01-01T00:00:00Z
 EWZ,10,Erewhon,Short Period,,1991-03-21T03:43:00Z,2001-07-02T14:38:00Z
 EXPV,10,Express,Short Period,,1995-10-25T03:58:00Z,1995-11-24T06:22:00Z
 EYCZ,10,Eyrewell,,,2012-03-20T21:00:01Z,2013-12-02T22:00:00Z
+EY2M,50,West Melton Geomagnetic Observatory,,,2024-04-04T16:00:00Z,9999-01-01T00:00:00Z
```

Comment:
please update station name to `West Melton Geomagnetic Observatory 2` or `West Melton 2 Geomagnetic Observatory`

Reply:
Amended